README.md

```
@@ -44,13 +44,28 @@ bin/kamal setup
-詳細は[デプロイガイド](./DEPLOY.md)を参照してください。
+詳細は[デプロイガイド](../../DEPLOY.md)を参照してください。
 
 ## 開発メモ
 
-## ログイン時のパスワード検証
+## 開発サーバーの立ち上げ方
+
+開発サーバーを立ち上げるには、以下のコマンドを実行します：
+
+```bash
+$ bin/dev
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)